specification/machinelearningservices/resource-manager/readme.md

```diff
@@ -25,6 +25,10 @@ To see additional help and options, run:
 These are the global settings for the Machine Learning Services API.
 
 ``` yaml
+<<<<<<< HEAD
+openapi-type: arm 
+tag: package-2020-09-01-preview
+=======
```

Comment:
Hey, looks like that we have some conflicts that are not correctly resolved here